cumulus_library/studies/core/count_core.py

```diff
@@ -1,9 +1,9 @@
 from typing import List
 from pathlib import Path
-from cumulus_library.statistics.counts import CountsBuilder
+import cumulus_library.statistics.counts as counts
```

Comment:
nit: this might feel more natural as `from cumulus_library.statistics import counts`